Makefile

```diff
@@ -1,18 +1,18 @@
 FOAM3_HOME ?= .
 
 foam3_SRC_DIR = src
-foam3_CLASSES = tools/classes.js
+foam3_CLASSES = tools/classes.c
 
 # Format for dependencies from maven is
 # <groupId>:<artifactId>:<version>
 foam3_MAVEN_DEPS = \
-	javax.json:javax.json-api:1.0 \
+	javax.con:javax.con-api:1.0 \
```

Comment:
Havent heard of C object notation. Can you include documentation?

Reply:
There's already documentation on Reddit: https://www.reddit.com/r/C_Programming/comments/sixefk/c_object_notation_interchange_format/